Backend/DataAccessLayer/Migrations/201903031732527_create-table-point.cs

```diff
@@ -0,0 +1,31 @@
+namespace DataAccessLayer.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class createtablepoint : DbMigration
+    {
+        public override void Up()
+        {
+            CreateTable(
+                "dbo.Points",
+                c => new
+                    {
+                        Id = c.Guid(nullable: false),

```

Comment:
Any indexing on this?

Reply:
I added indexing, but I'm not sure it's what you want. Please advise.

Reply:
:+1: